Sources/Fullscreen/NotificationCenter/NotificationCenterSectionHeaderView.swift

```diff
@@ -0,0 +1,107 @@
+
+    // MARK: - Internal properties
+
+    weak var delegate: NotificationCenterSectionHeaderViewDelegate?
+
+    // MARK: - Private properties
+
+    private var section: Int?
+
+    private lazy var titleLabel = Label(
+        style: .title3Strong,
+        withAutoLayout: true
+    )
+
+    private lazy var notificationDetailsView: PushNotificationDetailsView = {
+        let view = PushNotificationDetailsView(withAutoLayout: true)
+        view.addTarget(self, action: #selector(handleDetailsViewSelected), for: .touchUpInside)
+        return view
+    }()
+
+    private lazy var stackView: UIStackView = {
+        let stackView = UIStackView(arrangedSubviews: [titleLabel, notificationDetailsView])
+        stackView.axis = .vertical
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        return stackView
+    }()
+
+    private lazy var separatorView: UIView = {
+        let view = UIView(withAutoLayout: true)
+        view.backgroundColor = .tableViewSeparator
+        return view
+    }()
+
+    private lazy var stackViewTopConstraint = stackView.topAnchor.constraint(
+        equalTo: contentView.topAnchor
+    )
+
+    // MARK: - Init
+
+    override init(reuseIdentifier: String?) {
+        super.init(reuseIdentifier: reuseIdentifier)
+
+        contentView.backgroundColor = .bgPrimary
+        contentView.addSubview(stackView)
+        contentView.addSubview(separatorView)
+
+        NSLayoutConstraint.activate([
+            stackViewTopConstraint,
+            stackView.leadingAnchor.constraint(equalTo: contentView.leadingAnchor, constant: .mediumLargeSpacing),
+            stackView.trailingAnchor.constraint(equalTo: contentView.trailingAnchor, constant: -.mediumLargeSpacing),
+
+            separatorView.heightAnchor.constraint(equalToConstant: 1 / UIScreen.main.scale),
+            separatorView.leadingAnchor.constraint(equalTo: contentView.leadingAnchor),
+            separatorView.trailingAnchor.constraint(equalTo: contentView.trailingAnchor),
+            separatorView.bottomAnchor.constraint(equalTo: contentView.bottomAnchor),
+
+            contentView.bottomAnchor.constraint(equalTo: stackView.bottomAnchor),
+        ])
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    // MARK: - Methods
+
+    override func prepareForReuse() {
+        super.prepareForReuse()
+        configure(with: nil, inSection: 0)
+    }
+
+    func configure(with model: NotificationCenterSectionHeaderViewModel?, inSection section: Int) {
+        self.section = section
+
+        if let title = model?.title {
+            titleLabel.text = title
+            titleLabel.isHidden = false
+            stackViewTopConstraint.constant = .largeSpacing
+        } else {
+            titleLabel.isHidden = true
+            stackViewTopConstraint.constant = 0
+        }
+
+        notificationDetailsView.configure(with: model?.details)
```

Comment:
Should `notificationDetailsView` be hidden if `model?.details` is nil?

Reply:
It doesn't hurt to hide it if it's `nil`, I'll do that